ferrilata-devilline/Services/Extensions/AdminJsonAnalizer.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ferrilata_devilline.Services.Extensions
+{
+    public static class AdminJsonAnalizer
+    {
+        private class ExpectedFormat
+        {
+            [JsonProperty(Required = Required.Always)]
+            string version { set; get; }
+
+            [JsonProperty(Required = Required.Always)]
+            string name { set; get; }
+
+            [JsonProperty(Required = Required.Always)]
+            string tag { set; get; }
+
+            [JsonProperty(Required = Required.Always)]
+            List<object> levels { set; get; }
+
+            public ExpectedFormat()
```

Comment:
Explicit empty constructor is redundant.

Reply:
ok! If the Json parser will let me.